beep/tests/test_maccor_to_biologic_mb.py

```diff
@@ -55,14 +142,14 @@ def test_rest_step_conversion(self):
             "          <EndType>Voltage </EndType>"
             "          <SpecialType> </SpecialType>"
             "          <Oper>&gt;= </Oper>"
-            "          <Step>070</Step>"
+            "          <Step>02</Step>"
             "          <Value>4.4</Value>"
             "        </EndEntry>"
             "        <EndEntry>"
             "          <EndType>Voltage </EndType>"
             "          <SpecialType> </SpecialType>"
             "          <Oper>&lt;= </Oper>"
-            "          <Step>070</Step>"
+            "          <Step>02</Step>"
```

Comment:
I think this should be `002` The step number is zero padded to 3 digits

Reply:
good catch